packages/utils/scaffold.ts

```diff
@@ -0,0 +1,107 @@
+import chalk from "chalk";
+import * as j from "jscodeshift";
+import pEachSeries = require("p-each-series");
+import * as path from "path";
+
+import { IConfig, ITransformConfig } from "./modify-config-helper";
+import propTypes from "./prop-types";
+import astTransform from "./recursive-parser";
+import runPrettier from "./run-prettier";
+
+/**
+ *
+ * Maps back transforms that needs to be run using the configuration
+ * provided.
+ *
+ * @param	{Object} config 			- Configuration to transform
+ * @returns {Array} - An array with keys on which transformations need to be run
+ */
+
+function mapOptionsToTransform(config: any): string[] {
+	return Object.keys(config.webpackOptions).filter((k: string) => propTypes.has(k));
+}
+
+/**
+ *
+ * Runs the transformations from an object we get from yeoman
+ *
+ * @param	{Object} transformConfig 	- Configuration to transform
+ * @param	{String} action 			- Action to be done on the given ast
+ * @returns {Promise} - A promise that writes each transform, runs prettier
+ * and writes the file
+ */
+
+export default function runTransform(transformConfig: ITransformConfig, action: string): void {
+	// webpackOptions.name sent to nameTransform if match
+	const webpackConfig = Object.keys(transformConfig).filter((p: string) => {
```

Comment:
Could we create an interface for the object we send to `runTransform`?

Reply:
We already have `ITransformConfig`.